src/components/Message/types.ts

```diff
@@ -281,6 +282,8 @@ export type MessageUIComponentProps<
   renderText?: (
     text?: string,
     mentioned_users?: UserResponse<Us>[],
+    MentionComponent?: React.ComponentType<MentionProps<Us>>,
```

Comment:
I think it might be cleaner to remove this param if we change the API to this, since users can now also override the Mention component by specifying it inside `options.customMarkdownRenderers`.

Reply:
ahh right, good call on that, will remove